common/src/main/java/com/wynntils/screens/guides/ingredient/WynntilsIngredientGuideScreen.java

```diff
@@ -119,29 +116,7 @@ protected void renderTitle(PoseStack poseStack, String titleString) {
 
     private void renderTooltip(PoseStack poseStack, int mouseX, int mouseY) {
         if (hovered instanceof GuideIngredientItemStackButton guideGearItemStack) {
-            GuideIngredientItemStack itemStack = guideGearItemStack.getItemStack();
-
-            List<Component> tooltipLines = itemStack.getTooltipLines(McUtils.player(), TooltipFlag.NORMAL);
-            tooltipLines.add(Component.empty());
-
-            String unformattedName = itemStack.getIngredientProfile().getDisplayName();
-            if (Models.Favorites.isFavorite(unformattedName)) {
-                tooltipLines.add(Component.translatable("screens.wynntils.wynntilsGuides.itemGuide.unfavorite")
-                        .withStyle(ChatFormatting.YELLOW));
-            } else {
-                tooltipLines.add(Component.translatable("screens.wynntils.wynntilsGuides.itemGuide.favorite")
-                        .withStyle(ChatFormatting.GREEN));
-            }
-            tooltipLines.add(Component.translatable("screens.wynntils.wynntilsGuides.itemGuide.open")
-                    .withStyle(ChatFormatting.RED));
-            RenderUtils.drawTooltipAt(
-                    poseStack,
-                    mouseX,
-                    mouseY,
-                    0,
-                    tooltipLines,
-                    FontRenderer.getInstance().getFont(),
-                    true);
+            this.renderTooltip(poseStack, guideGearItemStack.getItemStack(), mouseX, mouseY);
```

Comment:
(Qodana here too? I would assume it is a bug in Qodana?)

Reply:
The method in question resides in the superclass, which is `Screen` in vanilla code. Maybe there is some kind of source mapping issue, since I assume method arguments are renamed during obfuscation.

And yes, the code is correct here too.